crates/bitwarden-threading/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "bitwarden-threading"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license-file.workspace = true
+keywords.workspace = true
+
+[features]
+wasm = ["dep:tsify-next", "dep:wasm-bindgen", "dep:js-sys"] # WASM support
```

Comment:
We're not using the feature or these dependencies at the moment, so we can probably remove them.

Reply:
Removed!